--- a/chrome/app/chrome_crash_reporter_client.cc
+++ b/chrome/app/chrome_crash_reporter_client.cc
@@ -134,7 +134,7 @@ bool ChromeCrashReporterClient::GetCrash
 void ChromeCrashReporterClient::GetProductInfo(ProductInfo* product_info) {
   CHECK(product_info);
 
-  const char* product_name = "Helium";
+  const char* product_name = "rac";
   *product_info =
       ProductInfo(product_name, version_info::GetHeliumVersionNumber(),
                   chrome::GetChannelName(chrome::WithExtendedStable(true)));
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Helium Authors
-COMPANY_SHORTNAME=The Helium Authors
-PRODUCT_FULLNAME=Helium
-PRODUCT_SHORTNAME=Helium
-PRODUCT_INSTALLER_FULLNAME=Helium Installer
-PRODUCT_INSTALLER_SHORTNAME=Helium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Helium Authors. All rights reserved.
-MAC_BUNDLE_ID=net.imput.helium
+COMPANY_FULLNAME=The rac Authors
+COMPANY_SHORTNAME=The rac Authors
+PRODUCT_FULLNAME=rac
+PRODUCT_SHORTNAME=rac
+PRODUCT_INSTALLER_FULLNAME=rac Installer
+PRODUCT_INSTALLER_SHORTNAME=rac Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The rac Authors. All rights reserved.
+MAC_BUNDLE_ID=me.kainoa.rac
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=S4Q33XPHB4
+MAC_TEAM_ID=
--- a/chrome/common/chrome_paths_mac.mm
+++ b/chrome/common/chrome_paths_mac.mm
@@ -32,7 +32,7 @@ char* ProductDirNameForBundle(NSBundle*
     product_dir_name = [product_dir_name_ns fileSystemRepresentation];
 
     if (!product_dir_name) {
-      product_dir_name = "net.imput.helium";
+      product_dir_name = "me.kainoa.rac";
     }
 
     // Leaked, but the only caller initializes a static with this result, so it
--- a/components/os_crypt/common/keychain_password_mac.mm
+++ b/components/os_crypt/common/keychain_password_mac.mm
@@ -34,8 +34,8 @@ namespace {
 // These two strings ARE indeed user facing.  But they are used to access
 // the encryption keyword.  So as to not lose encrypted data when system
 // locale changes we DO NOT LOCALIZE.
-const char kDefaultServiceName[] = "Helium Storage Key";
-const char kDefaultAccountName[] = "Helium";
+const char kDefaultServiceName[] = "rac Storage Key";
+const char kDefaultAccountName[] = "rac";
 
 // These values are persisted to logs. Entries should not be renumbered and
 // numeric values should never be reused.
