--- a/chrome/browser/resources/tab_search/tab_search_item.ts
+++ b/chrome/browser/resources/tab_search/tab_search_item.ts
@@ -228,8 +228,8 @@ export class TabSearchItemElement extend
 
     // Show chrome:// if it's a chrome internal url
     const protocol = new URL(normalizeURL(data.tab.url)).protocol;
-    if (protocol === 'chrome:' || protocol === 'helium:') {
-      this.$.secondaryTextInner.prepend(document.createTextNode('helium://'));
+    if (protocol === 'chrome:' || protocol === 'rac:') {
+      this.$.secondaryTextInner.prepend(document.createTextNode('rac://'));
     }
   }
 
--- a/content/public/common/url_constants.h
+++ b/content/public/common/url_constants.h
@@ -19,7 +19,7 @@ namespace content {
 inline constexpr char kChromeDevToolsScheme[] = "devtools";
 inline constexpr char kChromeErrorScheme[] = "chrome-error";
 inline constexpr char kChromeUIScheme[] = "chrome";  // Used for WebUIs.
-inline constexpr char kHeliumUIScheme[] = "helium";  // also used for webuis :3
+inline constexpr char kHeliumUIScheme[] = "rac";  // also used for webuis :3
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 inline constexpr char kChromeUIUntrustedScheme[] = "chrome-untrusted";
 inline constexpr char kViewSourceScheme[] = "view-source";
