--- a/components/helium_services/helium_services_helpers.cc
+++ b/components/helium_services/helium_services_helpers.cc
@@ -27,19 +27,6 @@ namespace helium {
 #define UPDATES_SUPPORTED
 #endif
 
-#if defined(UPDATES_SUPPORTED)
-constexpr std::string_view kUpdateBaseUrl = "https://updates.helium.computer/";
-constexpr const char kUpdatePath[] =
-#if BUILDFLAG(IS_MAC)
-    "mac/appcast-" CPU_ARCH_STR ".xml";
-#elif BUILDFLAG(IS_WIN)
-    "win/appcast.xml";
-#endif
-
-// Duplicated from helium_flag_choices.h to avoid importing it.
-constexpr const char kUpdateServerUrlCommandLine[] = "custom-update-server-url";
-#endif
-
 std::optional<GURL> GetValidUserOverridenURL(std::string_view user_url_) {
     if (user_url_.empty()) {
         return std::nullopt;
@@ -139,21 +126,8 @@ GURL GetSpellcheckURL(const PrefService&
 }
 
 GURL GetBrowserUpdateURL(const PrefService& prefs) {
-  if (!ShouldAccessUpdateService(prefs)) {
-    return GetDummyURL();
-  }
-
-#if defined(UPDATES_SUPPORTED)
-  const auto* command_line = base::CommandLine::ForCurrentProcess();
-  std::string baseUrl(kUpdateBaseUrl);
-  if (command_line->HasSwitch(helium::kUpdateServerUrlCommandLine)) {
-    baseUrl =
-        command_line->GetSwitchValueASCII(helium::kUpdateServerUrlCommandLine);
-  }
-  return GURL(baseUrl).Resolve(kUpdatePath);
-#else
+  // rac has no update feed yet. Helium's feed would replace rac with Helium.
   return GetDummyURL();
-#endif
 }
 
 GURL GetComponentUpdateURL(const PrefService* prefs) {
